posts/2021-08-30-sharing-data-with-fine-grained-control.md

```diff
@@ -0,0 +1,792 @@
+        ingredients {
+          name
+          quantity
+        }
+      }
+    }
+  }
+}
+```
+</details>
+
+We will receive results that look similar to the following:
+
+```json
+{
+  "data": {
+    "listRecipes": {
+      "Recipes": [
+        {
+          "id": "017b70eb-6d92-7039-4507-cc48eddb3c7c",
+          "name": "Sprinkles Cupcake",
+          "price": null,
+          "sku": null,
+          "recipeType": null,
+          "recipeYield": null,
+          "directions": null,
+          "ingredients": null
+        },
+        {
+          "id": "017b70ef-6e45-fd52-73f8-91c6862bdd68",
+          "name": "Red Velvet Cake",
+          "price": 5,
+          "sku": "ca001",
+          "recipeType": "cake",
+          "recipeYield": 1,
+          "directions": [
+            "Mix dry ingredients",
+            "Bake",
+            "Profit"
+          ],
+          "ingredients": [
+            {
+              "name": "All-purpose Flour",
+              "quantity": "453 grams"
+            },
+            {
+              "name": "Granulated Sugar",
+              "quantity": "680.3 grams"
+            }
+          ]
+        }
+      ]
+    }
+  }
+}
+```
+
+`Eve`'s permissions on the Sprinkles Cupcake recipe have not changed. `Eve` should still be able to see the recipe **name**.
+
+### Bob's Data Visibility
+
+If we run the same listRecipes query from `Bob`'s GraphQL Explorer, we will see all data for the Sprinkles Cupcake recipe, not just the name.
+
+```json
+{
+  "data": {
+    "listRecipes": {
+      "Recipes": [
+        {
+          "id": "017b70eb-6d92-7039-4507-cc48eddb3c7c",
+          "name": "Sprinkles Cupcake",
+          "price": 5.99,
+          "sku": "cc001",
+          "recipeType": "cupcake",
+          "recipeYield": 100,
+          "directions": [
+            "Mix dry ingredients",
+            "Bake",
+            "Let cupcakes cool for 20min",
+            "Make icing",
+            "Put icing on cupcakes",
+            "Profit"
+          ],
+          "ingredients": [
+            {
+              "name": "All-purpose Flour",
+              "quantity": "783.33 grams"
+            },
+            {
+              "name": "Granulated Sugar",
+              "quantity": "833 grams"
+            }
+          ]
+        },
+        {
+          "id": "017b70ef-6e45-fd52-73f8-91c6862bdd68",
+          "name": "Red Velvet Cake",
+          "price": 5,
+          "sku": "ca001",
+          "recipeType": "cake",
+          "recipeYield": 1,
+          "directions": null,
+          "ingredients": null
+        }
+      ]
+    }
+  }
+}
+```
+
+## Conclusion
+
+Vendia makes it easy, fast, and fun to share data with partners.  With the introduction of fine-grained control, Uni participants now have the added capability of sharing the right data with the right control with other parties. Review our [product documentation](https://www.vendia.net/docs/share/fine-grained-data-permissions) for more information.
```

Comment:
I like this conclusion paragraph a lot.  I'd try to keep terminology similar with the intro so the two are telling the same story with the same words. 

For example, "fine-grained control" here could be expanded to "fine-grained data access control" and used in both locations (in place of some of the other, competing terms).